generators/server/templates/pom.xml.ejs

```diff
@@ -137,7 +138,9 @@
         <%_ if (!skipClient) { _%>
         <sonar.testExecutionReportPaths>${project.testresult.directory}/jest/TESTS-results-sonar.xml</sonar.testExecutionReportPaths>
         <%_ } _%>
+        <%_ if (applicationType !== 'microservice') { _%>
```

Comment:
Testing with the value of skipClient (as done above on line 138) is more accurate I think, since you can generate e.g. a gateway or a monolith without the client side.